jme3-networking/src/main/java/com/jme3/network/service/rmi/MethodInfo.java

```diff
@@ -104,8 +103,8 @@ public static String methodToString( Method m ) {
     public static CallType getCallType( Method m ) {
         if( m.getReturnType() != Void.TYPE )
             return CallType.Synchronous;
-        if( m.getAnnotation(Oneway.class) != null )
-            return CallType.Asynchronous;
+        //if( m.getAnnotation(Oneway.class) != null )
+        //    return CallType.Asynchronous;

```

Comment:
Isn't this a rather big change?

Reply:
Not as far as I can tell, as I have not gotten an answer for where Oneway is even used. This is the only place in the entire JME where it is referenced, and this is only check for the class, not actually using it. So at this point, I have no idea why it is here, or where/who is using it for what.

Reply:
I thought I explained that already.  It was for USER CODE... so that they could write service code that could be used in SpiderMonkey that wasn't tied to SpiderMonkey's codebase.  

Say you wanted to write a general RMI style service for interacting with OAUTH or something... and you wanted to be able to target multiple deployments but still you want to be able to use asynchronous calls without having to include "ALL OF JME" just to have an Async annotation.  Then you could use Oneway which was a standard Java annotation.

Now it's not standard anymore so we can just "f*ck you" to these theoretical users.

Reply:
Thank you for the clarification @pspeed42, I did not fully understand that from our conversation on the forums. 

Seeing as there is not a standard annotation for this anymore, we can have our own, but then that would require the end user to include JME. Could we do a separate artifact that contains interfacing annotations to spider monkey that would be decoupled from JME?

Reply:
We already have our own: https://javadoc.jmonkeyengine.org/com/jme3/network/service/rmi/Asynchronous.html

...not really worth making a separate artifact.  We could also just loop through the annotations on the method and see if it has one called (fully qualified) "Oneway" using string comparison instead of class checks.  It's not perfect but I bet it won't be the ugliest thing we do to support JDK11.  (I wonder if this will be Java's last version?)

Reply:
OK, I think that is a good solution for now, I will perform a string check. That way we can keep backwards compatibility to an extent. 
(JDK 12 is out and about right now, I think we have more of this madness to come)

Reply:
OK, How does that look?

Reply:
@tlf30 @pspeed42 
I really do not get why you are not agree to add this dependency to `jme3-networking` module  ???
```
// https://mvnrepository.com/artifact/javax.jws/javax.jws-api
compile group: 'javax.jws', name: 'javax.jws-api', version: '1.1'
```

then we do not need to write any ugly code.

Reply:
That would improve performance, my understanding is that reflection can be really slow sometimes, so I don't really like it there. On the other hand I don't like adding an entire library for a single annotation. Perhaps @pspeed42 has an opinion, I think I am leaning towards this solution just from a performance standpoint.

Reply:
I haven't seen the change.  The link must have gotten lost in the other 20 messages.

...but A) reflection is not slow unless you do it badly.  b) I don't see how a string comparison would be "reflection".

Reply:
It does grab the annotations attached to the method. But considering we already have the Method object, I have to agree that this probably won't impact performance.

It does not matter to me which way we implement this. I think the string comparison is fine, but including the library is fine if we don't care about adding an extra library.

Reply:
If we used more than one class from it then including the extra library might be fine... but I feel uncomfortable doing it for one class for a feature that probably won't be commonly used.  I did it to be nice and because "it was there".  A string compare is a nice compromise and doesn't send any extra message in the dependency tree that we might do more advanced "WS" stuff that we don't.